models/db/setting.go

```diff
@@ -0,0 +1,222 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package db
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"code.gitea.io/gitea/modules/cache"
+	setting_module "code.gitea.io/gitea/modules/setting"
+	"code.gitea.io/gitea/modules/timeutil"
+	"code.gitea.io/gitea/modules/util"
+
+	"xorm.io/builder"
+)
+
+type ResourceSetting struct {
+	ID           int64              `xorm:"pk autoincr"`
+	GroupID      int64              `xorm:"index unique(key_repoid)"`               // to load all of some group's settings
+	SettingKey   string             `xorm:"varchar(255) index unique(key_groupid)"` // ensure key is always lowercase
```

Comment:
Sounds wrong. Do you mean
```suggestion
	GroupID      int64              `xorm:"index unique(key_groupid)"`               // to load all of some group's settings
	SettingKey   string             `xorm:"varchar(255) index unique(key_groupid)"` // ensure key is always lowercase
```
?